<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-03-28T11:06:57 IST" name="Default suite" finished-at="2022-03-28T11:07:20 IST" duration-ms="23534">
    <groups>
      <group name="Sanity">
        <method signature="SeleniumGrid1.failtest2()[pri:0, instance:seleniumGrid.SeleniumGrid1@5ace1ed4]" name="failtest2" class="seleniumGrid.SeleniumGrid1"/>
        <method signature="SeleniumGrid1.test1()[pri:0, instance:seleniumGrid.SeleniumGrid1@5ace1ed4]" name="test1" class="seleniumGrid.SeleniumGrid1"/>
      </group> <!-- Sanity -->
      <group name="Regression">
        <method signature="SeleniumGrid1.failtest2()[pri:0, instance:seleniumGrid.SeleniumGrid1@5ace1ed4]" name="failtest2" class="seleniumGrid.SeleniumGrid1"/>
        <method signature="SeleniumGrid1.test1()[pri:0, instance:seleniumGrid.SeleniumGrid1@5ace1ed4]" name="test1" class="seleniumGrid.SeleniumGrid1"/>
      </group> <!-- Regression -->
      <group name="Smoke">
        <method signature="SeleniumGrid1.failtest2()[pri:0, instance:seleniumGrid.SeleniumGrid1@5ace1ed4]" name="failtest2" class="seleniumGrid.SeleniumGrid1"/>
        <method signature="SeleniumGrid1.test1()[pri:0, instance:seleniumGrid.SeleniumGrid1@5ace1ed4]" name="test1" class="seleniumGrid.SeleniumGrid1"/>
      </group> <!-- Smoke -->
    </groups>
    <test started-at="2022-03-28T11:06:57 IST" name="Default test" finished-at="2022-03-28T11:07:20 IST" duration-ms="23534">
      <class name="seleniumGrid.SeleniumGrid1">
        <test-method is-config="true" signature="setupClass()[pri:0, instance:seleniumGrid.SeleniumGrid1@5ace1ed4]" started-at="2022-03-28T11:06:57 IST" name="setupClass" finished-at="2022-03-28T11:07:03 IST" duration-ms="6627" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupClass -->
        <test-method signature="failtest2()[pri:0, instance:seleniumGrid.SeleniumGrid1@5ace1ed4]" started-at="2022-03-28T11:07:04 IST" name="failtest2" finished-at="2022-03-28T11:07:11 IST" duration-ms="7210" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [] but found [Rahul Shetty Academy]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [] but found [Rahul Shetty Academy]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at seleniumGrid.SeleniumGrid1.failtest2(SeleniumGrid1.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- failtest2 -->
        <test-method signature="test1()[pri:0, instance:seleniumGrid.SeleniumGrid1@5ace1ed4]" started-at="2022-03-28T11:07:11 IST" name="test1" finished-at="2022-03-28T11:07:20 IST" duration-ms="9598" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
      </class> <!-- seleniumGrid.SeleniumGrid1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
